src/styles/variables.scss

```diff
@@ -0,0 +1,23 @@
+$primary-color: #007bff;
+$hover-color: #0056b3;
+$font-family: 'Roboto', sans-serif;
+$card-width: 200px;
+$star-size: 20px;
+// Colors
+$nav-link-color: #060b35;
+$nav-link-hover-color: #00acdc;
+$card-title-hover-color: #34568b;
+$button-bg: #00acdc;
+$button-hover-bg: #fff;
+$button-hover-text: #00acdc;
+
+// Sizes
+$card-width: 200px;
+$gap-horizontal: 46px;
+$gap-vertical: 48px;
+$paddings-vertical: 50px;
+$paddings-horizontal: 40px;
+$card-padding: 20px;
+
+// Fonts
+$font-family: Arial, sans-serif;
```

Comment:
The variable `$font-family` is declared twice with different values. Ensure only one declaration is present to avoid conflicts and unexpected behavior.